src/postData.js

```diff
@@ -0,0 +1,18 @@
+const xhr = new XMLHttpRequest();
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/6XXWCXn9VJxphtkrJMBV/scores/';
+
+export default function postData(nameValue, scoreValue) {
+  if (typeof nameValue.value === 'string') {
+    const params = `user=${nameValue.value}&score=${scoreValue.value}`;
+    xhr.open('POST', url, true);
+    xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
+    xhr.onreadystatechange = () => {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        alert(xhr.responseText);
+      }
+    };
+    xhr.send(params);
+  } else {
+    alert('Please enter a string only for name or a number only for score');
+  }
+}
```

Comment:
- [ ] Here as well, Use `arrow function` instead of using `function` keyword. We have supposed to use `ES6` syntax which improves the readability and it is best for callbacks as well.

- [ ] I would recommend using `async` and `await` with the `fetch` method to send and receive data. It is fully asynchronous and Promise based and it requires less code compared to XMLHttpRequest. you can know more about both `XMLHttpRequest` and `async-await` [here](https://betterprogramming.pub/from-xhr-to-fetch-with-async-await-on-the-finish-line-b021de1d226b).

Reply:
I have used fetch to get the data from the API